shopyo/modules/login/view.py

```diff
@@ -23,13 +25,18 @@
 )
 
 
+class LoginForm(FlaskForm):
+    user_id = StringField('user_id')
+    password = PasswordField('password')
+
```

Comment:
Since this is one of the first restructuration PRs, can you add the above in a forms.py then import `from modules.login.forms import LoginForm` ? Thanks

Reply:
Done!